tests/checkIfDeadTest.php

```diff
@@ -135,7 +134,7 @@ public function provideSanitizeURL() {
 				'http://www.silvercityvault.org.uk/index.php?a=ViewItem&key=SHsiRCI6IlN1YmplY3QgPSBcIkJyaWRnZXNcIiIsIk4iOjUyLCJQIjp7InN1YmplY3RfaWQiOiIyMCIsImpvaW5fb3AiOjJ9fQ%3D%3D&pg=8&WINID=1384795972907' ],
 			[ 'http://example.com/blue+light%20blue?blue%2Blight+blue%23foobar#foobar',
 				'http://example.com/blue+light%20blue?blue%2Blight+blue%23foobar' ],
-			[ 'http://www.musicvf.com/Buck+Owens+%2526+Ringo+Starr.art',
+			[ '://www.musicvf.com/Buck+Owens+%2526+Ringo+Starr.art',
```

Comment:
Best not to test 2 different things with one test. Can we keep this URL as it was for testing + handling and create a new test for malformed scheme?